.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER - Login user for docker hub
+# DOCKER_PASS - Login password for docker hub user
+version: 2.0
+
+jobs:
+  release-tag: # build for the master branch and tags
+    machine: true
+    working_directory: ~/addons-frontend-code
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: sudo apt-get update; sudo apt-get install pigz
+      - run: docker info
+        # Build the container, using Circle's Docker cache. Only use 1 image per
+        # day to keep the cache size down.
+      - run:
+          name: Build the container ( Next 3 steps )
+          command: IMG="image-$(date +%j).gz"; if [[ -e ~/addons-frontend-code/docker/$IMG ]]; then echo "Loading $IMG"; pigz -d -c ~/addons-frontend-code/docker/$IMG | docker load; fi
+      - run: >
+          printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n'
+          "$CIRCLE_SHA1"
+          "$CIRCLE_TAG"
+          "$CIRCLE_PROJECT_USERNAME"
+          "$CIRCLE_PROJECT_REPONAME"
+          "$CIRCLE_BUILD_URL"
+          > version.json
```

Comment:
We'll have to update the `Dockerfile` to copy this `version.json` file.

I suggest we had a `version.json` file in this project (with `{}` as content) and the following in the `Dockerfile`:

```
# Add `version.json` file for ops
COPY --from=build /app/version.json ./
```

That way, we'll end up with a similar image locally or in docker hub (via circle-ci).